framework/bin/d4j/d4j-info

```diff
@@ -57,6 +57,7 @@ use strict;
 use Constants;
 use Project;
 use Getopt::Std;
+use Carp qw(confess);
```

Comment:
Is the Carp module necessary? confess doesn't seem to be used.

Reply:
not sure how this ended up in here--never even heard of Carp before 🤷

Reply:
If it's not used, just delete it.